docs/ingestion/compaction.md

```diff
@@ -217,7 +217,7 @@ Druid supports two supported `inputSpec` formats:
 |-----|-----------|--------|
 |`segmentGranularity`|Time chunking period for the segment granularity. Defaults to 'null', which preserves the original segment granularity. Accepts all [Query granularity](../querying/granularities.md) values.|No|
 |`queryGranularity`|The resolution of timestamp storage within each segment. Defaults to 'null', which preserves the original query granularity. Accepts all [Query granularity](../querying/granularities.md) values.|No|
-|`rollup`|Whether to enable ingestion-time rollup or not. Defaults to 'null', which preserves the original setting. Note that once data is rollup, individual records can no longer be recovered. |No|
+|`rollup`|Whether to enable ingestion-time rollup or not. Defaults to 'null', which preserves the original setting. Once the data is rolled up, you can no longer recover individual records. |No|
```

Comment:
```suggestion
|`rollup`|Enable ingestion-time rollup or not. Defaults to 'null', which preserves the original setting. Once the data is rolled up, you can no longer recover individual records. |No|
```
I know it wasn't part of your change, but I try to avoid "whether". It tends to sound wordy. Also I think this should this be compaction-time rollup? cc: @maytasm , @vtlim

Reply:
I think this can be change to compaction-time rollup to be clear. But I think the original wording is with the assumption that compaction is a subtype of ingestion.